Weather App/view/SearchLocationViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SearchLocationViewController.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 23/03/22.
+//
+
+import UIKit
+
+class SearchLocationViewController: UIViewController {
+    
+    @IBOutlet private weak var locationSearchBar: UISearchBar!
+    @IBOutlet private weak var locationSuggestionTable: UITableView!
+    
+    static let id = "searchLocationViewController"
+    
+    public let searchLocationViewManager = SearchLocationViewModel()
+    public var currentWeatherViewModel = WeatherViewModel()
+    public var searchSuggestions : [SearchSuggestion] = []
```

Comment:
the class is internal and the properties are public?

Reply:
fixed